graphql/server.go

```diff
@@ -349,6 +349,7 @@ func (c *conn) handleMutate(in *inEnvelope) error {
 		go c.rerunSubscriptionsImmediately()
 
 		initial = false
+		go c.closeSubscription(id)
```

Comment:
Once we run a mutation, it makes a change to the data, and then returns the new data back to the client. By not closing the subscription, does that mean we were rerunning that mutation every time the data changed?

Reply:
We weren't rerunning the query because we return an error below.  The Rerunner exits immediately as soon as it gets an error that is not of type "RetrySentinelError".  The subscription would stick around though.

Reply:
Ahh that makes sense, nice catch!